doc/nrf/protocols/matter/end_product/platform_and_dmp.rst

```diff
@@ -0,0 +1,281 @@
+.. _ug_matter_platform_and_dmp:
+
+Matter platform and Derived Matter Product (DMP)
+###############################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Program summary
```

Comment:
People like images. Maybe we can create one for this section to show the whole process? We have something similar for certification here: https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/protocols/matter/end_product/certification.html#certification_process_overview

What do you think @peknis?

Reply:
Sure, images are often helpful and give information at a quick glance. If you decide to draw an image, please use Visio. Do not attach the original .vsd file, but the .svg in the PR. Make sure to have the original stored safely and when ready, ask me to save the original to the place where we store them for potential later adjustments.